core/src/test/scala/org/apache/spark/LocalSparkContext.scala

```diff
@@ -22,12 +22,37 @@ import org.scalatest.BeforeAndAfterAll
 import org.scalatest.BeforeAndAfterEach
 import org.scalatest.Suite
 
+import org.apache.spark.internal.Logging
 import org.apache.spark.resource.ResourceProfile
 
 /** Manages a local `sc` `SparkContext` variable, correctly stopping it after each test. */
-trait LocalSparkContext extends BeforeAndAfterEach with BeforeAndAfterAll { self: Suite =>
+trait LocalSparkContext extends Logging
+  with BeforeAndAfterEach with BeforeAndAfterAll { self: Suite =>
+
+  private var _conf: SparkConf = new SparkConf()
```

Comment:
I think `SparkConf` should have the default values for `master` and `appName`. So test suites extend it could use the SparkContext directly without any specific configurations when the test doesn't really care.

Reply:
OK